libtenzir/src/json_parser.cpp

```diff
@@ -42,14 +37,10 @@ auto ndjson_parser::parse(simdjson::padded_string_view json_line) -> void {
     }
     auto doc = *doc_it;
     TENZIR_ASSERT(not doc.current_location().error());
-    const auto source = std::string_view{
-      doc.current_location().value_unsafe(),
-      json_line.data() + json_line.size(),
-    };
     if (auto err = doc.error()) {
       diagnostic::warning("{}", error_message(err))
         .note("line {}", lines_processed_)
```

Comment:
I think we can still keep this more informative by printing the column as well. Maybe:

```suggestion
        .note("line {} column {}", lines_processed_, doc.current_location.value_unsafe()-json_line.data())
```

The same applies to the four changes below.

Reply:
Fair enough. The 2nd to last is the case where there was nothing parsed in the entire line anyways and the last one already tells you that it truncated N bytes at the end